components/productions/index/delete-modal.vue

```diff
@@ -3,15 +3,18 @@
     id="deleteModal"
     :ok-title="$t('productions.delete')"
     :cancel-title="$t('productions.cancel')"
+    cancel-variant="outline-secondary"
```

Comment:
The `cancel-variant` will probably be `outline-secondary` in 99% of our cases. So in line with `panel` and `panel-footer`, I'd suggest to add a custom `modal` component. It can just use `b-modal` internally and (for now) always set `cancel-variant` to `outline-secondary`, as well as set `hide-header`, and pass any other props through to `b-modal`.

Reply:
Will be implemented in next PR